lib/trino-parquet/src/test/java/io/trino/parquet/reader/TestPageReader.java

```diff
@@ -312,6 +312,77 @@ private static void assertPages(
         assertThat(pageReader.readPage()).isNull();
     }
 
+    @Test
+    public void testPageSizeLimit()
```

Comment:
package private https://trino.io/docs/current/develop/tests.html#conventions-and-recommendations